collection-scripts/collect_service_logs

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+function collect_service_logs {  ## Takes a node role input (master or worker) and a list of services
+    selector=${1}
+    shift
+
+    PIDS=()
+    DIR_PATH=${SERVICE_LOG_PATH}/${role}s
+    echo "WARNING: Collecting one or more service logs on ALL linux $1 nodes in your cluster. This could take a large amount of time." >&2
```

Comment:
I think this will fire all the time (even if your only collecting on masters, which isn't something we want).

Reply:
Now I only print the warning if --role=workers